cli-plugins/apps/reference/workload-create-apply.hbs.md

```diff
@@ -1,11 +1,14 @@
-# Tanzu apps workload apply
+# tanzu apps workload apply
 
-`tanzu apps workload apply` is a command used to create and update workloads that are deployed in a cluster through a supply chain.
+`tanzu apps workload apply` is a command used to create and update workloads that are deployed in a
+cluster through a supply chain.
 
-The `tanzu apps workload apply` and `tanzu apps workload create` commands have the same behavior and flags with the following exceptions:
+The `tanzu apps workload apply` and `tanzu apps workload create` commands have the same behavior and
+flags with the following exceptions:
 
 - The `tanzu apps workload create` command fails if a workload with the same name preexists on the target cluster.
-- the `update-strategy` flag is only applicable to the `tanzu apps workload apply` command. The `update-strategy` flag is not applicable to the `tanzu apps workload create` command.
+- The `update-strategy` flag is only applicable to the `tanzu apps workload apply` command.
```

Comment:
- The `update-strategy` flag is _only_ supported by `tanzu apps workload apply` (because `create` doesn't support updating existing workloads).

Reply:
ok, changed